src/gfnff/dmat.f

```diff
@@ -0,0 +1,45 @@
+! This file is part of xtb.
+!
+! Copyright (C) 2019-2020 Stefan Grimme
+!
+! xtb is free software: you can redistribute it and/or modify it under
+! the terms of the GNU Lesser General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! xtb is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU Lesser General Public License for more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+cccccccccccccccccccccccccccccccccccccccccccccc
+c density matrix
+c C: MO coefficient
+c X: scratch
+c P  dmat
+cccccccccccccccccccccccccccccccccccccccccccccc
+
+      subroutine dmat(ndim,focc,C,P)
```

Comment:
Can `dmat` from `xtb_scc_core` used instead?

Reply:
yes, definitely!

Reply:
`xtb_scc_core` is now used